scripts/install-worker.sh

```diff
@@ -191,6 +197,8 @@ net.bridge.bridge-nf-call-iptables = 1
 net.ipv4.ip_forward = 1
 EOF
 
+sudo mv "${TEMPLATE_DIR}/00-runtime-slice.conf" /etc/systemd/system/containerd.service.d/00-runtime-slice.conf
```

Comment:
I think this should be done at runtime, so `containerd` is only configured this way when it's the `CONTAINER_RUNTIME`

Reply:
I think that makes sense, can you confirm that Containerd isn't started before it's configured in bootstrap?

Reply:
Wouldn't that be the place where it'd be most likely that the Containerd service had already run? If anything it needs to be at the top, but we still need to confirm it hasn't run yet.

Reply:
I've done this and tidied up the command order slightly for consistency. I agree that the if status logic looks like it needs reviewing; AFAIK installing the RPM will not enable or start the units.

Reply:
> AFAIK installing the RPM will not enable or start the units.

Correct, I tested that yesterday.

---

You'll need to change this line, though; because `TEMPLATE_DIR` doesn't exist in `bootstrap.sh`. I suggested inlining the dropin because otherwise you'll have to place the file in an intermediate location during build.

Reply:
This is the install script change, would you like both inlined? Or we do use a temporary location for the conditional files so I could use that for the bootstrap change?

Reply:
Should have commented on the `bootstrap.sh` thread, this change is outdated. This is the only one that's necessary: https://github.com/awslabs/amazon-eks-ami/pull/1051#discussion_r1087104085